src/hooks/useKeyboardShortcut/index.ts

```diff
@@ -68,10 +70,11 @@ const useKeyboardShortcut = (
             event.preventDefault();
           }
           cbRef.current();
+          sendEvent({ name: "Used Shortcut", keys: JSON.stringify(keys) });
```

Comment:
I wonder if we should process the keys more into a specific format... But if having the JSON in a string isn't problematic and doesn't make the New Relic queries difficult, then that's okay too!

Reply:
Good point! I have found it a little tricky to query json in new relic. I've replaced this with a string and a function to format the string